compiler/rustc_mir_dataflow/src/value_analysis.rs

```diff
@@ -258,6 +258,9 @@ pub trait ValueAnalysis<'tcx> {
                 // They would have an effect, but are not allowed in this phase.
                 bug!("encountered disallowed terminator");
             }
+            TerminatorKind::TailCall { .. } => {
+                // FIXME(explicit_tail_calls): determine if we need to do something here (probably not)
```

Comment:
oof, we'll need to investigate the users of `handle_call_return` to see how to best handle these, but I presume we'll want to treat it similar to a return.